src/sonic-yang-models/yang-models/sonic-mirror-session.yang

```diff
@@ -154,14 +154,9 @@ module sonic-mirror-session {
                         "Destination port configuration for port mirroring(SPAN).";
                 }
 
-                leaf-list src_port {
-                    type union {
-                        type leafref {
-                            path "/port:sonic-port/port:PORT/port:PORT_LIST/port:name";
-                        }
-                        type leafref {
-                            path "/lag:sonic-portchannel/lag:PORTCHANNEL/lag:PORTCHANNEL_LIST/lag:name";
-                        }
+                leaf src_port {
+                    type string {
+                        length 1..256;
```

Comment:
Is 256 enough for src_port?

Reply:
modified to 2048. should allow configuring upto 200 ports in single session.